index.php

```diff
@@ -107,6 +107,22 @@ function videoMIMEsubtype() {
 		<meta property="og:description" content="<?php echo $description; ?>" />
 		<meta property="al:web:url" content="https://openings.moe/?video=<?php echo $s_filename; ?>" />
 
+		<!-- Google Stuff -->
+		<meta name="google-site-verification" content="J8ZBk2wWSOvLneswCoGGHmrwRPBy9HbX7RHxAGFgz4E" />
+		<script type="application/ld+json">
+			{
+				"@context" : "http://schema.org",
+				"@type" : "WebSite",
+				"url" : "https://openings.moe/",
+				"potentialAction" : {
+					"@type" : "SearchAction",
+					"target" : "https://openings.moe/search/?s={search_term_string}",
```

Comment:
Did you forget to add something to your commit? I assume /search/index.php should exist as part of this PR?